File: C++/Matrix/Transposematrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& A) {
        vector<vector<int>>ResultMatrix(A[0].size(),vector<int>(A.size(),0));
        
        for(int i=0;i<A.size();i++)
        {
            for ( int j=0;j<A[0].size();j++)
            {
                ResultMatrix[j][i] = A[i][j];
            }
        }
        
        return ResultMatrix;
    }
};

File: Python/Matrix/Transpose Matrix.py
class Solution(object):
    def transpose(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        ResultMatrix=[[0*j for j in range(len(A))]for i in range(len(A[0]))]
        for i in range(len(A)):
            for j in range(0,len(A[0])):
                ResultMatrix[j][i] = A[i][j]
        
        return ResultMatrix
                
                
